docs/how-to/system-optimization/mi300a.rst

```diff
@@ -122,6 +122,51 @@ This section describes performance-based settings.
 
        transparent_hugepage=always
 
+* **Increase the amount of allocatable memory**
+
+  By default, when using a device allocator via HIP, it is only possible to allocate 96GB out of 
+  a possible 128GB of memory on the MI300A. This limitation does not affect host allocations.
```

Comment:
Should 96 GiB and 128 GiB be used for consistency?

Reply:
Sure. I can make this change.